src/dsp_tools/utils/xml_upload.py

```diff
@@ -320,12 +334,14 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
         uploaded because there is an error in it
     """
 
+    # Validate the input XML file
+    validate_xml_against_schema(xml_file_as_path_or_parsed=xml_file_as_path_or_parsed)
```

Comment:
```suggestion
    validate_xml_against_schema(xml_file_as_path_or_parsed)
```

Reply:
I have a general question: I thought it would be better style to always use keyword arguments if possible ("explicit is better than implicit"). Would you say that this is not a good rule?

Reply:
In this case I would say that explicity doesn't add any value but complexity instead. It think it is becuase of the long parameter name. As the parameter name and the parameter value are called exactly the same, it can as well be left out. What do you think?

Reply:
I have shortened the parameter name (per recommendation of Balduin), but left in place the keyword. Now it looks fine :-) Thanks for your thoughts.